docs/antrea-network-policy.md

```diff
@@ -265,6 +266,30 @@ spec:
       enableLogging: true
 ```
 
+#### ACNP for Pod-level isolation in Namespaces
```

Comment:
suggestion: ACNP for complete Pod isolation in all selected Namespaces

As I am not totally sure what "Pod-level isolation" means

Reply:
Done